rpcserver.go

```diff
@@ -6815,6 +6815,12 @@ func (r *rpcServer) UpdateChannelPolicy(ctx context.Context,
 			minTimeLockDelta)
 	}
 
+	if req.TimeLockDelta > uint32(MaxTimeLockDelta) {
```

Comment:
nit: can use `if {} else {}` since both of these will never be true at the same time